staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -144,7 +145,12 @@ func createHandler(r rest.NamedCreater, scope *RequestScope, admit admission.Int
 				return
 			}
 
-			obj, err = scope.FieldManager.Update(liveObj, obj, managerOrUserAgent(options.FieldManager, req.UserAgent()))
+			fieldManager, err := defaultFieldManager(obj, options.FieldManager, req.UserAgent())
+			if err != nil {
+				scope.err(fmt.Errorf("failed to find valid fieldManager (Create for %v): %v", scope.Kind, err), w, req)
```

Comment:
When I read this error, I assumed that this would also be returned if no fieldmanager was found. Maybe we can improve:
`error while looking for default fieldManager (Create for %v): %v`?